fuse/data/README.md

```diff
@@ -309,13 +324,15 @@ The following operators are useful when implementing a common pipeline:
 * OpToTensor - convert many different types to PyTorch tensor
 * OpOneHotToNumber - convert one-hot encoding vectors into numbers
 
+[**Tabular augmentations**](ops/ops_aug_tabular.py)
+
+* OpAugOneHot - Apply an augmentation for an one-hot encoding vector
```

Comment:
Added the Op here. 

All the other changes are result of some linter.. not on purpose but looks good.